docs/spec_draft.md

```diff
@@ -223,6 +224,48 @@ Produces a `result` tensor from a constant `value`.
 // %3: [(0.0, 1.0), (2.0, 3.0)]
 ```
 
+## stablehlo.cosine
+
+`stablehlo.cosine(operand) -> result`
+
+### Semantics
+
+Performs element-wise cosine operation on `operand` tensor and produces a `result` tensor.
+
+### Attributes
```

Comment:
We do not need an Attribute for this op.

Reply:
here and below

Reply:
Fixed